### DISCUSSION TITLE
Progress Report + Outlook: June 2022

### DISCUSSION THREAD

**Discussion by mkurz** [playframework/playframework#11313] [Announcements]
Hi everyone!

I've been thinking about writing this post for a few weeks now, basically to keep you up to date and because there are some things that are on my mind that I finally want to share with the community. (This post is a bit personal and not coming from the steering committee)
Since I am the one that initiated the move to Open Collective and most people see me as the "Play guy" being in charge, I often get mentioned in GitHub comments, being asked when the next release will happen, when this or that bug gets fixed or being told someone is waiting for a pull request to be reviewed or to be merged. I guess some people might be wondering why things take a bit longer right now or why I was a bit idle in the last weeks, so I want to explain what I am up to and how I plan to proceed the next couple of weeks and the upcoming months.

Let's get straight to the point:
* We are officially on Open Collective since around 6 month now
* We have a couple of premium sponsors now and many individuals giving smaller amounts which, summed up, equal one premium sponsorship
* The monthly recurring *net* income from Open Collective (meaning the actual amount that can be payed out, after fees retained by Open Collective and other transaction fees) does not yet fully compensate a senior full-time dev/project manager
* However, the sponsorship is enough to allow me to take the risk and finally go all in and start working full-time on the Play Framework, starting with July, looking for more sponsors along the way

I also want to share a couple of things about my personal situation and how things evolved for me:
* My time unfortunately was very limited since September last year (the time when talks started to hand over Play to someone else), that is because I still had contracts to fulfill, my wife was (and until end of June still is) finishing her (last) master's degree, plus we have two small kids, which consume a lot of our time and energy, especially because my wife often is busy as well...
* Until now, I probably could invest 20%, sometimes maybe up to to 40%, of my energy into Play. I never was satisfied with that, my head often was blocked with my many other things that were going on (other projects, taking care of kids, other non-Play things, and so on). I often had the feeling Play falls short.
* However, the good news is from July on I will have much more time and, like said, will finally gradually start working full-time on Play, finally being able to invest most (soon all) of my energy into this awesome open source project
* Starting with July, I am confident I can spend at least 3, much more likely 4, sometimes 5 days working on Play (5 is the target, but can't promise that)
* At the latest in September there should be nothing holding me back anymore, finally working full-time on Play, calling it my job (no more contracts, not more studies, kids in kindergarten, no more moving into a new flat, etc.). Until that, I also want to attract a couple of more premium sponsors.
* What about June? Like said, my wife still has some things to do and our youngest still isn't in kindergarten, so I also take care of our kids a lot right now. On top of that we spontaneously moving into a new flat currently (within the same house however).
* Since I don't want to burn out, I decided to step down a bit in June, which means I will be around, but probably don't have time to do more than some commenting.
* However, because Play isn't just me but also the fantastic people from rest of the steering committee, there are always other people around! (Thanks guys!)

Enough from me.
So, it has been half a year that Play moved to a community lead project, announcing our Open Collective page on Twitter just before Christmas, let's do a quick recap what happened so far. If you follow the Play repositories, you probably have seen that, even with everyones limited time we
* cut a couple of releases (including important security fixes)
  * Not talking solely about the main Play repository, but also sub projects like play-ebean, play-json, play-ws,...
* improved our build and release infrastructure
* cleaned up most repositores (still not done yet ;)
* I mean, basically made the source code and tests compile again and not fail anymore, since many tests basically were broken since last summer...
* reviewed and merged a lot of pull requests from many awesome contributors, including some important pull requests to help Play and its sub projects supporting Scala 3 (still waiting on akka-http...)
* like mentioned, acquired premium sponsors
* doing a couple of live streams (which is something we plan to do more in future, with better quality ;)
* many, many other (small?) things that I can't remember all right now.

Also, let me mention, besides obvious things that you see because they are public, there are other things going on which are hidden to the public, but still consume quite some time, like spending hours in Zoom conversations with (potential) premium sponsors or organizing the steering committee. (Again, Play is not just me, but also very nice people from the steering committee that spend time working on the project)

So, that's it for now. Personally, I do have big plans for Play and look forward to the future.

I plan to write such a progress update every month or at least each quarter now, just to be more transparent and keep everyone up to date, especially our premium sponsors.

One last thing: What I plan do as well, starting from beginning of July, is that I will update my GitHub status each week, to let you know what I am up to (working on Play, on holiday, etc.). I was thinking about that for a while already. You can subscribe to my GitHub profile here: https://github.com/mkurz

Thanks again everyone for your patience and your support, be it financially, by contributing or by just continuing using Play. It's highly appreciated!

Best,
Matthias

**Comment by schemata-issimo** [comment]
Hi Matthias,

Really appreciate the update! I feel like so many organisations derive so much value from play framework, and sometimes they just need a little nudge to spur them to act on sending just the tiniest amount of revenue in response, the whole open collective movement seems pretty new but certainly timely after seeing what happened with OpenSSL. Personally I'm still kicking revenue into gear, but receive a huge, huge amount of value from Play, it's so great to see the constant updates and maintenance, looking forward to joining the sponsor list once we get a budget for this kind of thing.

Great job protecting yourself from burnout and thanks for leading the way.

All the best
Nick

**Reply by mkurz** [reply]
Thanks for your support Nick, highly appreciated! If you think about a premium sponsorship and have some questions or you (or someone else) want(s) to talk personally, feel free to reach out to me (contact at pla..ork.com)
